ui/app/styles/core/forms.scss

```diff
@@ -184,16 +184,16 @@ label {
   }
 }
 
-.select:not(.is-multiple)::after {
+.select:not(.is-multiple)::after,
+.select:not(.is-multiple)::before {
   border-color: $black;
   border-width: 2px;
   margin-top: 0;
   transform: translateY(30%) rotate(-45deg);
 }
 
 .select:not(.is-multiple)::before {
-  @extend .select:not(.is-multiple)::after;
```

Comment:
Dart Sass didn't like this - we didn't end up using that, but I also thought it wasn't super necessary so I added the selector above.